microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -295,124 +388,146 @@ See also [DevTools: Focus Mode UI](https://github.com/MicrosoftEdge/MSEdgeExplai
 
 
 <!-- ====================================================================== -->
-<!-- ## DevTools Tooltips -->
+## DevTools Tooltips
 <!-- present in 96, 98 -->
 
+Needs content.
```

Comment:
This was also documented in What's New for 91 here: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/04/devtools#learn-about-devtools-with-informative-tooltips

Reply:
added